allennlp_models/tagging/models/crf_tagger.py

```diff
@@ -205,6 +212,7 @@ def forward(
         loss : `torch.FloatTensor`, optional
             A scalar loss to be optimised. Only computed if gold label `tags` are provided.
         """
+        ignore_loss_on_o_tags = ignore_loss_on_o_tags or self.ignore_loss_on_o_tags
```

Comment:
I think this should be:

```suggestion
        ignore_loss_on_o_tags = ignore_loss_on_o_tags if ignore_loss_on_o_tags is not None else self.ignore_loss_on_o_tags
```

Reply:
Thanks @epwalsh for your review! I fixed it.